js/jquery.mapael.js

```diff
@@ -290,9 +290,16 @@
                             delete plots[deletedPlots[i]];
                         }
                     }
+                // Delete ALL plots if deletedPlots is set to "all"
+                } else if (deletedPlots === "all") {
+                    $.each(plots, function(id, elem) {
+                        fnRemoveElement(elem);
+                    });
+                    // Empty plots array
+                    plots.length = 0;
```

Comment:
Oh right, I don't know why I supposed that `plots` or `links` were array! They are object! I will change it.
For array, setting length to 0 will shrink the array size automatically.